hotspot/agent/src/share/classes/sun/jvm/hotspot/utilities/Hashtable.java

```diff
@@ -63,7 +63,7 @@ protected static long hashSymbol(byte[] buf) {
     int len = buf.length;
     // Emulate the unsigned int in java_lang_String::hash_code
     while (len-- > 0) {
-      h = 31*h + (0xFFFFFFFFL & buf[s]);
+      h = 31*h + (0xFFL & buf[s]);
```

Comment:
`Byte.toUnsignedInt()` would be clearer.

Reply:
yes, but that would also be different to upstream jdk11u